spec/services/atmosphere/cloud/satisfy_appliance_spec.rb

```diff
@@ -440,20 +441,20 @@
         let(:vmt1_1) { create(:virtual_machine_template, tenants: [t1])}
         let(:vmt2_1) { create(:virtual_machine_template, tenants: [t1])}
         let(:vmt2_2) { create(:virtual_machine_template, tenants: [t2])}
-        let(:wf_set_2) { create(:appliance_set, appliance_set_type: "workflow")}
+        let(:wf_set_2) { create(:appliance_set, appliance_set_type: "workflow", user: u)}
```

Comment:
Line is too long. [89/80]<br>Space missing inside }.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.